FlagsmithClient.cs

```diff
@@ -261,6 +262,19 @@ public async Task<Trait> SetTrait(string identity, string key, object value)
             }
         }
 
+        public async Task SetTraits(string identity, Dictionary<string, object> traits)
+        {
+            ArrayList traitDataObjects = new ArrayList();
+
+            foreach (KeyValuePair<string, object> trait in traits)
+            {
+                traitDataObjects.Add(new { identity = new { identifier = identity }, trait_key = trait.Key, trait_value = trait.Value });
+            }
+
+            string url = configuration.ApiUrl.AppendPath("traits/bulk");

```

Comment:
Maybe we should add a note that this endpoint is not part of edge and will not exist in v2